vi/cs-230-recurrent-neural-networks.md

```diff
@@ -0,0 +1,677 @@
+**Recurrent Neural Networks translation** [[webpage]](https://stanford.edu/~shervine/teaching/cs-230/cheatsheet-recurrent-neural-networks)
+
+<br>
+
+**1. Recurrent Neural Networks cheatsheet**
+
+&#10230; Cheatsheet về mạng neural hồi quy
+
+<br>
+
+
+**2. CS 230 - Deep Learning**
+
+&#10230; CS 230 - Deep Learning
+
+<br>
+
+
+**3. [Overview, Architecture structure, Applications of RNNs, Loss function, Backpropagation]**
+
+&#10230; [Tổng quan, Kết cấu kiến trúc, Ứng dụng của RNNs, Hàm mất mát, Lan truyền ngược]
+
+<br>
+
+
+**4. [Handling long term dependencies, Common activation functions, Vanishing/exploding gradient, Gradient clipping, GRU/LSTM, Types of gates, Bidirectional RNN, Deep RNN]**
+
+&#10230; [Xử lí các phụ thuộc dài hạn, Các hàm kích hoạt phổ biến, Vanishing/exploding gradient, Gradient clipping, GRU/LSTM, Các loại cổng, RNN hai chiều, RNN xâu]
```

Comment:
I suggest use translation in this [sheet by DL book VN](https://docs.google.com/spreadsheets/d/1Hcd4AqR7Xzd08Ws0gwhZ-LX6XNzJKgCDy0VcFc3D66c/edit#gid=0)

```suggestion
&#10230; [Xử lí các phụ thuộc dài hạn, Các hàm kích hoạt phổ biến, tiêu biến/bùng  gradient, Gradient clipping, GRU/LSTM, Các loại cổng, RNN hai chiều, Deep RNN]
```

Reply:
I think it's a good idea but, with **Vanishing/exploding gradient**, we should keep it in English to protect the meaning of those concepts.